src/attr/__init__.pyi

```diff
@@ -65,7 +66,12 @@ class AttrsInstance(Protocol):
 
 # _make --
 
-NOTHING: object
+# Typed as enum member so that Literal[NOTHING] will work.
+@enum.Enum
+class _NothingType:
```

Comment:
The enum name is part of the repr, right? Can we name it something nicer, like `Defaults` or at least `NothingType` (without the underscore)?

Reply:
Will do!

Reply:
Or would you like me to override `__repr__` so it's just `NOTHING`?

Reply:
Good question. @hynek ?

Reply:
Ah I didn't notice it's just the stub. Does it need to be just the stub, though?

Reply:
When I wrote it, it needed to be just the stub because of 2.7 support... [but I just saw it was recently dropped](https://www.attrs.org/en/stable/changelog.html#backwards-incompatible-changes). Excellent!

Now it can be changed to be the actual code. Now I'd recommend changing `__repr__` to be `'NOTHING'`, and making the class `_NothingType` again, to show that it's an implementation detail.

Reply:
Sounds good

Reply:
i understanding nothing (lol) of what y'all are talking about, but a repr of `NOTHING` sounds great